src/pretix/base/metrics.py

```diff
@@ -0,0 +1,150 @@
+import redis
+
+ns_prefix = "pretix_metrics_"
```

Comment:
I believe ``REDIS_KEY_PREFIX`` is clearer as a prefix is a well-defined concept while a namespaces does not exist in redis-world.